modules/apps/29-fee/ibc_module.go

```diff
@@ -224,7 +224,7 @@ func (im IBCModule) OnAcknowledgementPacket(
 		return sdkerrors.Wrapf(err, "cannot unmarshal ICS-29 incentivized packet acknowledgement: %v", ack)
 	}
 
-	packetID := channeltypes.NewPacketId(packet.SourceChannel, packet.SourcePort, packet.Sequence)
+	packetID := channeltypes.NewPacketId(packet.DestinationChannel, packet.DestinationPort, packet.Sequence)
```

Comment:
This line should stay as it is. Since it is executing on Source chain we want to identify packets using source channel/port

Reply:
`OnRecvPacket` needs to change on line 203 to reference destination port/channel because it is being executed on destination chain

Reply:
Sorry. This was late in an evening blunder. I meant to update `OnRecvPacket`. Casually introducing bugs into the codebase :sweat_smile: